docker/Dockerfile-osc-base

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+
+MAINTAINER Arvind Nadendla <arvindn05@gmail.com>
+
+ENV http_proxy http://proxy-chain.intel.com:911
+ENV https_proxy https://proxy-chain.intel.com:912
+
+RUN yum update -y && \
+yum install -y wget && \
+wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u161-b12/0da788060d494f5095bf8624735fa2f1/jre-8u161-linux-x64.rpm && \
+yum localinstall -y jre-8u161-linux-x64.rpm && \
```

Comment:
do we need to hardcode the jre version? can this be part of the configuration?

Reply:
I need to figure out better approach. There is no installer command for oracle java jre in CentOS.